feat(azure): nRF Cloud Location Services are now mandatory (#844)
NordicSemiconductor/asset-tracker-cloud-docs: docs/azure/GettingStarted/nRFCloudLocationServices.rst

.. _azure-nrf-cloud-location-services:

nRF Cloud Location Services
###########################

> nRF Cloud Location Services are an essential component of any asset tracking solution and nRF Asset Tracker makes heavy use of them.

Reviewer: ```suggestion
nRF Cloud Location Services is an essential component of any asset tracking solution and nRF Asset Tracker uses these services frequently.
```
